app/src/main/java/com/mathewsachin/fategrandautomata/accessibility/ScriptRunnerUserInterface.kt

```diff
@@ -33,6 +33,29 @@ class ScriptRunnerUserInterface(val Service: ScriptRunnerService) {
 
     val windowManager = Service.getSystemService(Context.WINDOW_SERVICE) as WindowManager
 
+    private val metrics: DisplayMetrics get() {
+        val res = DisplayMetrics()
+
+        windowManager.defaultDisplay.getRealMetrics(res)
+
+        return res
+    }
+
+    val mediaProjectionMetrics: DisplayMetrics get() {
```

Comment:
Can you add a KDoc explaining what this is used for? My assumption is that it's used to only take landscape screenshots and recordings.

Reply:
Yeah, the name has `mediaProjection` in it and there's a comment inside saying about using only landscape. KDoc feels redundant, still added.